docs/theming.md

```diff
@@ -185,9 +185,170 @@ extension MyViewController {
+
+Themers will eventually be deprecated and deleted. Theming extensions are the recommended
+replacement. This migration guide will cover the typical migration path from Themer usage to
+Theming extensions.
+
+In general, every component's `Themer` targets will gradually be replaced by a single `Theming`
+extension target. This includes:
+
+- ColorThemer
+- FontThemer
+- ShapeThemer
+- TypographyThemer
+- Component Themers (e.g. CardThemer)
+
+Some Theming extensions are still in Beta. To add a Beta Theming extension to your project you will
+need to follow the instructions in the [Beta components](../contributing/beta_components.md)
+documentation.
+
+Some components do not have a Theming extension yet. We are prioritizing the remaining Theming
+extensions in https://github.com/material-components/material-components-ios/issues/7172.
+
+#### Typical migration
+
+When migrating from a subsystem Themer to a Theming extension the first thing to understand is that
+a Theming extension will apply *all* of the available subsystems to a given component. If you were
+previously relying on the ability to apply only one subsystem (e.g. Color) to a component, please
+file a
+[feature request](https://github.com/material-components/material-components-ios/issues/new/choose)
+with a code snippet of your existing use case.
+
+Generally-speaking, the migration from a subsystem Themer to a Theming extension will involve the
+following steps:
+
+1. Update your dependencies.
+2. Update your imports.
+3. Changes to your code.
+
+##### Update your dependencies
+
+In order to use theming extensions you'll need to first follow the
+[beta component instructions](../../contributing/beta_components.md).
+
+When a component has a theming extension it will always be available as a `Theming` target
+alongside the component. For example:
+
+In CocoaPods:
+
+```ruby
+// Old
+pod 'MaterialComponents/Cards'
+pod 'MaterialComponents/Cards+ColorThemer'
+
+// New
+pod 'MaterialComponents/Cards'
+pod 'MaterialComponents/Cards+Theming'
+```
+
+In Bazel:
+
+```ruby
+// Old
+  deps = [
+      "//components/schemes/Cards",
+      "//components/schemes/Cards:ColorThemer",
+  ],
+
+// New
+  deps = [
+      "//components/schemes/Cards",
+      "//components/schemes/Cards:Theming",
+  ],
+```
+
+##### Update your imports
+
+Replace any Themer import with the component's Theming import:
+
+<!--<div class="material-code-render" markdown="1">-->
+#### Swift
+
+```swift
+// Old
+import MaterialComponents.MaterialCards_ColorThemer
+
+// New
+import MaterialComponents.MaterialCards_Theming
+```
+
+#### Objective-C
+
+```objc
+// Old
+#import <MaterialComponents/MaterialCards+ColorThemer.h>
+
+// New
+#import <MaterialComponents/MaterialCards+Theming.h>
+```
+<!--</div>-->
+
+##### Changes to your code
+
+Replace any Themer code with the equivalent use of a component's theming extension. Each Themer's
+equivalent Theming extension API is described in the Themer's API documentation.
```

Comment:
I'm not sure I'm following this last sentence. Did you mean "...in the Component's API documentation"?